src/libs/PolicyUtils.ts

```diff
@@ -1205,6 +1206,50 @@ function canModifyPlan(policyID?: string) {
     return !!policy && isPolicyAdmin(policy);
 }
 
+function getAdminsPrivateEmailDomains(policyID?: string) {
+    if (!policyID) {
+        return [];
+    }
+    const policy = getPolicy(policyID);
+    if (!policy) {
+        return [];
+    }
+
+    const adminDomains = Object.entries(policy.employeeList ?? {}).reduce((domains, [email, employee]) => {
+        if (employee.role === CONST.POLICY.ROLE.ADMIN) {
+            domains.push(Str.extractEmailDomain(email).toLowerCase());
+        }
+        return domains;
+    }, [] as string[]);
+
+    const ownerDomain = Str.extractEmailDomain(policy.owner).toLowerCase();
+
+    return [...new Set([...adminDomains, ownerDomain])].filter((domain) => !isPublicDomain(domain));
+}
+
+function getMostFrequentEmailDomain(policyID: string, acceptedDomains: string[]) {
+    const policy = getPolicy(policyID);
```

Comment:
Same here, if the `getMostFrequentEmailDomain()` is used in a view, then any data referenced inside the function must also come from the view or else the view is not reactive.